test/integration/developer_can_edit_profile_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.